frontend/App.js

```diff
@@ -1,21 +1,53 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import * as React from 'react';
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { MaterialCommunityIcons, Ionicons, FontAwesome5 } from '@expo/vector-icons';
+import { DiagnosticsTab, DataTab, HomeTab, AmbianceTab, Start } from './homeTabs';
+import SettingsTab from './settingsTab';
 
-export default function App() {
+function HomeTabs() {
   return (
-    <View style={styles.container}>
-      <Text>Sleep thingy holder text</Text>
-      <StatusBar style="auto" />
-    </View>
+    <Tab.Navigator initialRouteName="Home" screenOptions={({ route }) => ({
+      tabBarIcon: ({ focused, color, size }) => {
+        switch (route.name) {
+          case "Diagnostics":
+            return <MaterialCommunityIcons name="hospital" size={size} color={color} />;
+          case "Data":
+            return <MaterialCommunityIcons name="waveform" size={size} color={color} />;
+          case "Home":
+            return <Ionicons name="ios-globe-outline" size={size} color={color} />;
+          case "Ambiance":
+            return <FontAwesome5 name="lightbulb" size={size} color={color} />;
+          case "Settings":
+            return <Ionicons name="settings-outline" size={size} color={color} />;
+          default:
+            return;
```

Comment:
Here is where the tab icons are implemented. I did this in the Tab.Navigator inside the screenOptions, but could easily be implemented at an individual Tab.Screen level.

Reply:
I got a warning when I ran it and only the icon for Ambiance showed up:
"Warning: Failed prop type: invalid prop `name` of value `x` supplied to `Icon`" 
where x is waveform, ios-globe-outline, and settings-outline

Reply:
@courtneylau Did you run npm install to install the necessary modules for the icons?